packages/config/tests/api/tests.js

```diff
@@ -323,76 +332,52 @@ test('In integration dev mode, integration specified in config is returned and b
   t.assert(config.integrations)
   t.assert(config.integrations.length === 1)
   t.assert(config.integrations[0].slug === 'abc-integration')
-  t.assert(config.integrations[0].has_build === true)
+  t.assert(config.integrations[0].has_build === false)
```

Comment:
This change might be necessary but looks a little out of place. Flagging just in case it was a mistake